lib/assertions.js

```diff
@@ -226,34 +226,54 @@ export default class DOMAssertions {
 
     if (value instanceof RegExp) {
       let result = value.test(actualValue);
-      let expected = `Element ${this.targetDescription} has attribute "${name}" with value matching ${value}`;
-      let actual = actualValue === null
-        ? `Element ${this.targetDescription} does not have attribute "${name}"`
-        : `Element ${this.targetDescription} has attribute "${name}" with value ${JSON.stringify(actualValue)}`;
+      let expected = `Element ${
+        this.targetDescription
+      } has attribute "${name}" with value matching ${value}`;
+      let actual =
+        actualValue === null
+          ? `Element ${
+              this.targetDescription
+            } does not have attribute "${name}"`
+          : `Element ${
+              this.targetDescription
+            } has attribute "${name}" with value ${JSON.stringify(
+              actualValue
+            )}`;
```

Comment:
@scalvert this looks like unrelated changes 🤔 

I'm assuming you used prettier here, but I'm not sure if the result in this case is actually prettier than before... 😄

Reply:
Damn. It’s vscode. I format on save for some repos and not for others. It’s maddening. Will fix.